equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+		},
+		"updated_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Updated by Date and Time",
+		},
+		"deleted_by": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Key",
+		},
+		"deleted_by_full_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Full Name",
+		},
+		"deleted_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Email Address",
+		},
+		"deleted_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by Date and Time",
+		},
+	}
+}
+
+func readOrderSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"purchase_order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Purchase order number",
+		},
+		"billing_tier": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Billing tier for connection bandwidth",
+		},
+		"order_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Identification",
+		},
+		"order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Reference Number",
+		},
+	}
+}
+
+func readAccountSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_number": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Account Number",
+		},
+		"account_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Account Name",
+		},
+		"org_id": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Customer organization identifier",
+		},
+		"organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Customer organization name",
+		},
+		"global_org_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization identifier",
+		},
+		"global_organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization name",
+		},
+		"global_cust_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global Customer organization identifier",
+		},
+	}
+}
+
+func readNotificationSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"type": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Notification Type",
+		},
+		"send_interval": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Send interval",
+		},
+		"emails": {
+			Type:        schema.TypeList,
+			Computed:    true,
+			Description: "Array of contact emails",
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+		},
+	}
+}
+
+func readErrorAdditionalInfoSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"property": {
```

Comment:
Hmm. 
Are Property/Reason part of a log stream we are including in the Terraform resource?
My first impression on seeing this field is that it might be an error embedded in the resource that Terraform should just error on instead of portraying as resource attributes. 

What's the API spec reference for this?

Reply:
Yes, in certain scenarios; a connection can stuck in intermediate state with errors, fulfilled with errors. These errors are transparent to customer and are exposed using these elements.